MAUI/PDF-Viewer/Lock-Unlock.md

```diff
@@ -73,7 +73,7 @@ void LockCircleAnnotations()
 
 ## Lock the selected annotation
 
-To lock the selected annotation, access the selected annotation instance and set the `IsLocked` property of the annotation to `true`. The selected annotation instance may be obtained from the `AnnotationSelected` event. The following example explains how to lock the selected annotation in a PDF document.
+To lock the selected annotation, access the selected annotation instance and set the `IsLocked` property of the annotation to `true`. The selected annotation instance may be obtained from the [AnnotationSelected](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_AnnotationSelected) event. The following example explains how to lock the selected annotation in a PDF document.
```

Comment:
<img width="319" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/45b1d4fc-3d64-4ee1-8d44-b29eaa4b8cd7">

It is missing in some of the places

Reply:
Updated sir.

Reply:
Still it is unresolved.

<img width="340" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/7f3f3e00-8c8d-482f-8817-b2383fe8cb39">